sdk/core/core-https/package.json

```diff
@@ -96,7 +96,8 @@
     "form-data": "^3.0.0",
     "tslib": "^2.0.0",
     "https-proxy-agent": "^5.0.0",
-    "uuid": "^8.3.0"
+    "uuid": "^8.3.0",
+    "@azure/core-asynciterator-polyfill": "^1.0.0"
```

Comment:
I'd still prefer for this to go on `core-client` because I don't want to force all consumers of `core-https` to need to worry about this. 

I don't think Low Level Clients use async iterators? /cc @joheredi @bterlson

Reply:
@xirzec I ok to it to core-client. But even without adding it to core-client, if I use core-client, Wouldn't I automatically get it from core-https? The SDK users don't have to explicitly refer to core-https at all. So, what is the advantage of adding it to core-client? (On a side note, if you do want me to add the dependency to core-client, would you still want to keep the dependency on core-https? or drop it?)

Reply:
Basically, I don't think it should be a dependency of `core-https`, because there are scenarios where someone might want to depend on `core-https` but not need it. 

I want to keep `core-https` as small as possible.

Reply:
Done. Removed the dependency from core-https and added it to core-client

Reply:
Sorry I missed this but strong +1 what was already done (having the dep in core-client and not core-https).